package.json

```diff
@@ -1,8 +1,12 @@
 {
   "name": "@reedsy/express-ws",
-  "version": "5.0.0-reedsy-2.0.0",
+  "version": "5.0.0-reedsy-3.0.0",
   "description": "WebSocket endpoints for Express applications",
-  "main": "index",
+  "main": "index.cjs",
```

Comment:
I don't think this is used if [`exports` is defined](https://nodejs.org/api/packages.html#package-entry-points):

> If both ["exports"](https://nodejs.org/api/packages.html#exports) and ["main"](https://nodejs.org/api/packages.html#main) are defined, the ["exports"](https://nodejs.org/api/packages.html#exports) field takes precedence over ["main"](https://nodejs.org/api/packages.html#main) in supported versions of Node.js.

Reply:
It was [added in v12.7.0](https://nodejs.org/api/packages.html#exports), which was EOL in 2022. I think we're good on the support front.